portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase

```diff
@@ -170,6 +170,89 @@ definition {
 			key_position2 = "3");
 	}
 
+	@description = "Verify that a Form can be created with Spanish as the default portal language"
+	@priority = "5"
+	test CanBeCreatedWithSpanishAsDefaultLanguage {
```

Comment:
@leticiachiu, can you please verify with Diogo if this test can be added to the Acceptance Suite?

This is an important test which has already helped Forms Team prevent bugs. :wink: